meta-lmp-bsp/conf/machine/include/lmp-mfgtool-machine-custom.inc

```diff
@@ -86,3 +86,17 @@ KERNEL_CLASSES:mx8qm-generic-bsp = " kernel-lmp-fitimage "
 UBOOT_SIGN_ENABLE:mx8qm-generic-bsp = "1"
 UBOOT_CLASSES:mx8qm-generic-bsp = "uboot-fitimage"
 IMAGE_BOOT_FILES:mx8qm-nxp-bsp  = "imx-boot u-boot.itb boot.itb"
+
+# STM32MP1
+INITRAMFS_IMAGE:stm32mp1common = "initramfs-ostree-lmp-recovery"
+DISTRO_FEATURES:remove:stm32mp1common = "pam"
```

Comment:
Why are we dropping pam here? Not an issue, but good to have at least a comment saying why.